session/bootstrap.go

```diff
@@ -365,6 +365,17 @@ const (
 		oldReadLease bigint(20) NOT NULL DEFAULT 0,
 		PRIMARY KEY (tid)
 	);`
+	// CreateAnalyzeOptionsTable stores the analyze options used by analyze and auto analyze.
+	CreateAnalyzeOptionsTable = `CREATE TABLE IF NOT EXISTS mysql.analyze_options (
+		table_id BIGINT(64) NOT NULL,
+        sample_num BIGINT(64),
```

Comment:
Seems that line370 is tab and this line is space.

Reply:
Fixed.